### PR TITLE
support multiple files

### PR DESCRIPTION
**Pull request by Konboi**
The raw profile can't receive multiple report files, so I fixed supporting multiple report files.

### CODE REVIEW

File: launchable/test_runners/raw.py [hunk 1]
```diff
@@ -150,8 +150,12 @@ def parse_json(test_result_file: str) -> Generator[CaseEventType, None, None]:
                 test_path, duration_secs, CaseEvent.STATUS_MAP[status],
                 case['stdout'], case['stderr'], created_at)
 
-    if not test_result_file.endswith('.xml'):
-        client.parse_func = parse_json
+    for test_result_file in test_result_files:
+        if not test_result_file.endswith('.xml'):
+            client.parse_func = parse_json
```
Comment (shibuiwilliam) [inline launchable/test_runners/raw.py:155]: [question]
Don't we have to determine if the file ends with `.json`?
Or is it filtered beforehand?

Comment (Konboi) [reply launchable/test_runners/raw.py:155]: I found out. It's a historical reason.
We support only JSON format at first. After that, we started to support XML file format.
So, if the report file isn't XML, we set a json format parser.

https://github.com/launchableinc/cli/pull/505

**Review by shibuiwilliam — APPROVED** [review]
Only one comment.

File: launchable/test_runners/raw.py (hunk 1 above)
Comment (Konboi) [reply launchable/test_runners/raw.py:155]: We only support JSON and XML, so we don't need to check an extension is `.json` when it isn't `.xml`.

**Pull request merged**